src/utils/logUtils.ts

```diff
@@ -0,0 +1,34 @@
+import { logger } from '@navikt/next-logger'
+import { AxiosError } from 'axios'
+
+const UUID =
```

Comment:
Egentlig ikke så viktig, men synes det er litt vanskelig å lese den hardkodede UUID-regexen. FNR er jo greit og forståelig, men finnes det noen biblioteker vi bruker andre steder vi kunne brukt her, spesielt for standard UUID form?

Reply:
Godt innspill! Har ikke sett om det finnes bibliotek for det